frontend/src/components/ScheduleCreation.vue

```diff
@@ -437,9 +437,11 @@ watch(
           data-testid="dashboard-set-availability-toggle"
         />
       </div>
-      <alert-box @close="scheduleCreationError = ''" v-if="scheduleCreationError">
-        {{ scheduleCreationError }}
-      </alert-box>
+      <alert-box
+        v-if="scheduleCreationError"
+        :title="scheduleCreationError"
+        @close="scheduleCreationError = ''"
```

Comment:
Should this be set to `null` instead of empty quote?

Reply:
Oh, I thought I grabbed all alert-boxes. Good catch!